CMakeLists.txt

```diff
@@ -199,5 +200,5 @@ ENDIF(BUILD_TESTING)
 
 PKG_CONFIG_APPEND_LIBS(${PROJECT_NAME})
 IF(NOT INSTALL_PYTHON_INTERFACE_ONLY)
-  INSTALL(FILES package.xml DESTINATION share/${PROJECT_NAME})
+  INSTALL(FILES package.xml DESTINATION share/${PROJECT_NAME_UNDERSCORE})
```

Comment:
Is this really a hard requirement anywhere ? 

This kind of non-standardization really gives me more work, to adapt all my tooling (in robotpkg & my buildfarm) and write recipes that will be able to understand that this file is in another place… So I'd really like to avoid that.

Reply:
The point is that this repo is installing a partial ros_package from:
```
./ros/sot_talos_balance
```
Should we add another package.xml inside this sub-folder ? IMHO I do not think so.
To make it simpler I would move data, launch, urdf, and yaml at the root position, and install them directly with the package.xml.
On bionic it seems that the necessity to have underscore for ROS packages has been lifted.
We need to check if this is the case for xenial. If not then we need to have underscore for package name. 
Anyway right now, package.xml is installed in install/share/sot-talos-balance while the ros/sot_talos_balance directory is installed in install/share/sot_talos_balance.
This commit is just making this coherent.

We definitely need to simplify sot_talos_balance.

Reply:
Ok, so we don't know if this is a hard requirement or not yet, I'll make some tests.

I suggest to consider this as a separate issue, and not to change that in this PR.

Reply:
Having no package.xml in sot_talos_balance make the integration tests in sot_talos_balance/test fails.
And this not coherent from a ros viewpoint.